lib/thrift/binary/framed/server.ex

```diff
@@ -8,10 +14,45 @@ defmodule Thrift.Binary.Framed.Server do
           | {:max_seconds, non_neg_integer}
           | {:tcp_opts, :ranch_tcp.opts()}
           | {:ssl_opts, [Thrift.Transport.SSL.option()]}
+          | {:transport_opts, :ranch.opts()}
 
   @type server_opts :: [server_option]
 
-  @spec start_link(module, 1..65_535, module, [server_option]) :: GenServer.on_start()
+  @doc """
+  Starts the server using the specified handler module.
+
+  The following server options can be specified:
+
+  `:worker_count`: The number of acceptor workers used to take requests.
```

Comment:
acceptor workers to accept on the socket, different processes are used to handle requests